mne/io/base.py

```diff
@@ -1997,6 +2004,16 @@ def _write_raw(fname, raw, info, picks, fmt, data_type, reset_range, start,
 
         pos_prev = pos
 
+    if raw.annotations is not None:
+        start_block(fid, FIFF.FIFFB_MNE_ANNOTATIONS)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MIN, raw.annotations.onset)
+        write_int(fid, FIFF.FIFF_MNE_BASELINE_MAX, raw.annotations.duration)
```

Comment:
> Okay, so could we use FIFF_MNE_ANNOTATION_DURATION = FIFF_MNE_BASELINE_MIN in constants. Is there a risk that this overlap will cause trouble? Isn't annotations purely an attribute of raw and baseline for epochs and such?

It would probably be okay. We are more or less doing that here. But why not just add the duration to the onset to get the tmax and save that, and undo it on read, as I suggested? It's less name mangling to trade off with an arithmetic operation.

> Other option would be just introducing a couple of new constants. There are still plenty available.

I'd rather not introduce new constants unless we really need to. Even though right now it seems like plenty are available, adopting the attitude that plenty are available is a good way to make it so that no more are soon...